scripts/load-model.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
A similar script exists in the `ncpi-api-fhir-service` repo: https://github.com/ncpi-fhir/ncpi-api-fhir-service/blob/master/scripts/load_model.sh

Reply:
I've moved these scripts to a directory outside this repo to avoid duplication.